class.jetpack.php

```diff
@@ -1502,6 +1502,7 @@ public static function get_active_plan() {
 
 		if ( in_array( $plan['product_slug'], $premium_plans ) ) {
 			$supports[] = 'akismet';
+			$supports[] = 'simple-payments';
```

Comment:
Is this the right spot for this check, since this feature is used for both wpcom and Jetpack? @dzver do you have any suggestions here? I believe we do have plan capabilities defined elsewhere.

Reply:
It's a very common thing when implementing a new plan or a new plans-related endpoint to add a new way of defining features. We have many of these spread across the codebases. This $supports array is just one of many such implementations. It's hard to recommend one of the other implementations as superior. If you decide to refactor the whole thing, we can do a hangout with some other devs who worked on plans and agree on an official recommendation.

It doesn't make me happy to see this but the code is consistent with itself and I think is fine.

Reply:
Thanks @dzver, 🤔we'll keep this for now, but I think it'll be worth revisiting later.

Reply:
This was not my first option when implementing this, but since `simple-payments` is not a full Jetpack module (it's loaded by `modules-extras.php`) we can't use the automatic mapping of plan features to modules (using the `/sites/{site_id}/features` endpoint), or the automatic deactivation check on `check_plan_deactivate_modules`.  I had to treat it as an exception.

If we refactor SP into a complete Jetpack module, we can remove it from the exception list. But I think that exceeds the scope of this PR

I have a diff (D14013-code) that adds SP as a plan feature.